FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+        Boolean ShouldMatchWord { get; set; };
+        
+        /// Gets or sets the state of whether all matches are highlighted. 
+        /// Returns TRUE if all matches are highlighted, FALSE otherwise.
+        /// Note: Changes to this property take effect only when StartFind, FindNext, or FindPrevious is called. 
+        /// Preferences for the session cannot be updated unless another call to the StartFind function on the server-side is made. 
+        /// Therefore, changes will not take effect until one of these functions is called.
+        Boolean ShouldHighlightAllMatches { get; set; };
+        
+        /// Sets this property to hide the default Find UI. 
+        /// You can use this to hide the default UI so that you can show your own custom UI or programmatically interact with the Find API while showing no Find UI. 
+        /// Returns TRUE if hiding the default Find UI and FALSE if using showing the default Find UI.
+        /// Note: Changes to this property take effect only when StartFind, FindNext, or FindPrevious is called. 
+        /// Preferences for the session cannot be updated unless another call to the StartFind function on the server-side is made. 
+        /// Therefore, changes will not take effect until one of these functions is called.
+        Boolean SuppressDefaultFindDialog { get; set; };
+    };
+
+   runtimeclass CoreWebView2Find : [default]ICoreWebView2Find {}
+
+    /// Interface providing methods and properties for finding and navigating through text in the web view.
+    /// This interface allows for finding text, navigation between matches, and customization of the find UI.
+    [com_interface("staging=ICoreWebView2StagingFind")]
+    [ms_owner("core", "maxwellmyers@microsoft.com")]
+    [availability("staging")]
+    interface ICoreWebView2Find 
+    {
+        [completed_handler("")]
+        /// Initiates a find using the specified configuration asynchronously.
+        /// Displays the Find bar and starts the find operation. If a find session was already ongoing, it will be stopped and replaced with this new instance.
+        /// If called with an empty string, the Find bar is displayed but no finding occurs. Changing the configuration object after initiation won't affect the ongoing find session.
+        /// To change the ongoing find session, StartFindAsync must be called again with a new or modified configuration object.
+        /// StartFind supports, HTML, PDF, and TXT document queries. In general this api is designed for text based find sessions.
+        //// If you start a find session programmatically on another file format that doesnt have text fields, the find session will try to execute but will fail to find any matches. (It will silently fail)
+        /// Note: The asynchronous action completes when the UI has been displayed with the find term in the UI bar, and the matches have populated on the counter on the find bar. 
```

Comment:
Is the completion of the IAsyncAction the point at which calls to Next()/Previous()/Stop() methods become allowed?  What happens if I call Next() before this IAsyncAction  completes, is it the same "silently fail" as when there's no active session?